documentation-support/src/main/java/tech/picnic/errorprone/documentation/BugPatternExtractor.java

```diff
@@ -93,55 +91,16 @@ private static <T extends AnnotationValue> T doCast(AnnotationValue value, Class
     return (T) value;
   }
 
-  @AutoValue
-  @JsonDeserialize(as = AutoValue_BugPatternExtractor_BugPatternDocumentation.class)
-  abstract static class BugPatternDocumentation {
```

Comment:
I validated that the generated files in `error-prone-contrib/target/docs/*.json` are unchanged.